src/main/java/com/jagrosh/jmusicbot/commands/dj/SeekToCmd.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.commands.dj;
+
+import java.time.Duration;
+
+import javax.sound.midi.Track;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+
+/**
+ *
+ * @author John Grosh <john.a.grosh@gmail.com>
+ */
+public class SeekToCmd extends DJCommand 
+{
+    public SeekToCmd(Bot bot)
+    {
+        super(bot);
+        this.name = "seekto";
+        this.help = "seeks to the specified position in the currently playing song";
+        this.arguments = "<position>";
+        this.aliases = bot.getConfig().getAliases(this.name);
+        //this.bePlaying = true;
+    }
+
+    @Override
+    public void doCommand(CommandEvent event) 
+    {
+        String index = "";
+        try
+        {
+            index = event.getArgs();
+            if(index.indexOf(':') == -1){
+                throw new NumberFormatException("Wrong format. Example: 5:18 (up to hours only).");
```

Comment:
Should not fail, arithmetically divide the number into hours, minutes, etc.

If you wanted it to fail, use IllegalArgumentException.
NumberFormatException is for stuff like "0b7f" on binary, "3F" on dec,  or "0xGGCZ" on hex.